src/test/scala/com/linkedin/feathr/offline/FeatureGenIntegTest.scala

```diff
@@ -275,6 +278,99 @@ class FeatureGenIntegTest extends FeathrIntegTest {
     res.head._2.data.show(100)
   }
 
+  /**
+   * Test with non SWA feature with preprocessing and an application config writing to an output directory.
+   */
+  @Test(enabled = false)
```

Comment:
Enable it?

Reply:
It's like the previous test case it's based on. Since it has redis output so it actually will throw. So it's disabled. It's mostly used for debugging for devs.